docs/design/descheduler.md

```diff
@@ -0,0 +1,115 @@
+# DeScheduler
+
+@[Thor-wl](https://github.com/Thor-wl); Dec 25th, 2021
+
+## Motivation
+As what [Issue1777](https://github.com/volcano-sh/volcano/issues/1777) describes, **Rescheduling** is important for the 
+following reasons:
+* Unbalanced resource utilization due to unreasonable scheduling strategies and dynamic changes in jobs' lifecycle.
+* Node status changes such as add/remove nodes, pod/node taint/affinity changes.
+
+In order to rebalance the cluster resource utilization among nodes, introduce **DeScheduler** to achieve these goals:
+* Rescheduling pods based on real resource utilization instead of request resource.
+* Support custom configured rescheduling strategies.
```

Comment:
Add the inscope and out of scope to let users know what will be done in this project.

Reply:
Roadmap added to the `Implementation Plan` chapter